test/commands/link.test.ts

```diff
@@ -63,4 +66,26 @@ describe('link', () => {
     const linked_components = fs.readJSONSync(linked_components_file);
     expect(linked_components).to.have.property('hello-world', component_path);
   });
+
+  it('list linked components', async () => {
```

Comment:
It would be ideal to follow the same directory structure in the `tests` folder that we do for `src`. We have different test files for different commands such as `platforms`, `environments`, and `secrets`, so we should add one for `link` now too. Then each link command would have a different file in that folder

Reply:
Sounds great! I'll update that